### ISSUE TITLE
In some cases (1.4.0+) caldera-grid.css is not loading

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#754]

**Comment by Shelob9** [comment]
If you are experiencing this issue -- form fields are suddenly not responsive or form fields are not the right width, as a workaround, please add this to your theme or child theme's functions.php

```
	add_action( 'wp_enqueue_scripts', function(){
		wp_enqueue_script( 'cf-grid-styles', CFCORE_URL . 'assets/css/caldera-grid.css' );
		wp_enqueue_script( 'cf-form-styles', CFCORE_URL . 'assets/css/caldera-form.css' );
	});
```

**Comment by Shelob9** [comment]
This was fixed during 1.4.4
